README.md

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
- [ ] Kindly format your README using this [Microverse template](https://github.com/microverseinc/readme-template). Note that all the sections are required except for the Tests, Deployment, and Live demo sections if you do not have them.

- [ ] Kindly add more detailed instructions on how to set up and run your app locally, ie cloning the repo, installing packages, and starting the app.

- [ ] You are yet to delete all boilerplate files. Kindly delete the `public/robots.txt` file.